server.js

```diff
@@ -71,6 +71,15 @@ module.exports = function(options) {
     uri: process.env.APPLICATION_URI
   });
 
+  server.register(require("hapi-auth-bearer-token"), function(err) {});
```

Comment:
incredibly, ridiculously unlikely, considering the module versions won't change unless manually changed